### ISSUE TITLE
Any wizard panel update breaks builder in version > 4.7.8 [BUG] 

### ISSUE THREAD

**Issue by gregbown** [formio/formio.js#2389]
## Environment
- Hosting type
  - [ x] Local deployment
    - Version: 
- Formio.js version: 4.9.0-beta.2 - 4.8.0
- Frontend framework: NodeJS
- Browser: Chrome
- Browser version: 79.0.3945.130

## Steps to Reproduce
1. Create new wizard with text input, radio input, etc
2. Edit on any child element and its event data will reflect the changes
3. Edit the parent panel and no changes are reflected in the event data
4. Any changes made to a panel itself do not persist. If I change the panel title, it is only reflected in the builder while I remain on that panel. Moving from one panel to the next will reveal the changes in the builder do not persist.
5. On any attempt to change any settings in a top level panel, the builder breaks silently and I can no longer add any child elements.  Child elements will show in builder only, not persist and not be reflected in data..

[DEMO](https://stackblitz.com/edit/formio-builder-zw4dkm)

## Expected behavior
One should be able to change a panel attribute such as title
Editing a panel should not break ability to build out the form.

## Observed behavior
Any edits to a panel attribute, property etc. break builder.
Rolling back to formio.js 4.7.8 corrects these issues

## Note
I am using the code style shown [here](https://formio.github.io/formio.js/app/examples/custombuilder.html)
where the builder is passed in from then(builder ...). I believe using the factory method in FormBuilder.js
```
Formio.builder(document.getElementById('builder'), {}, {...})
.then(function(builder) {
  builder.on('saveComponent', function() {
    ...
  });
});
```

I do not have an explanation why version 4.8.1 works [here](https://formio.github.io/formio.js/app/builder) other than maybe slightly different instantiation method of builder and form render. Here the builder is a global returned by new Formio.FormBuilder
```
var builder = new Formio.FormBuilder(...)
builder.instance.on('saveComponent', onBuild);
builder.instance.on('editComponent', onBuild);
builder.instance.ready.then(onReady);
```

Please elaborate on the correct style of creating the builder if indeed this is the issue. I do see that when using the factory method for creating a new builder there is no builder.instance.form, there is only builder.form
Thank you

**Comment by jhen1422** [comment]
@gregbown 
I've tried reproducing using 4.8.0 and I have been unsuccessful. 

1 Create a new wizard
2 Add child components to the panel page - as mentioned changes made persist
3 Changing page panel (like title) reflects in the builder when I navigate between pages without any issues. 

Is there something I'm missing here?

**Comment by gregbown** [comment]
@jhen1422 Understood there must be some nuance in how the form is updated. I am using builder change event and the form instance setter when the form change event has components. See the demo I added.  As soon anything is done to the top level panel, the builder remains broken. 
Please advise... Thank you so much for looking into this Jeriah

**Comment by gorjupleon** [comment]
@jhen1422 @gregbown  Had the same issue, but I found a workaround solution:
```
builder.on('saveComponent', () => builder.schema);
```
As long as the builder.schema getter is called on the `change` or `saveComponent` event, even if nothing is done with it, it seems to update the panel normally. I assume the getter does something that changes the behaviour, but I didn't dig into the source code any further.

**Comment by gregbown** [comment]
@gorjupleon Thank you for passing this along! I verified this simple call [does in fact correct the issue](https://stackblitz.com/edit/formio-builder-ja6tkl?file=index.js), how strange.

**Comment by gregbown** [comment]
@travist This is fixed with 4.10.2.  Thank you!
Panel changes no longer break form in wizard mode. Also, calling schema getter is no longer needed and the infinite loop is also fixed.
